server/index.js

```diff
@@ -33,14 +37,8 @@ app.use(express.static(path.join(__dirname, '../client/dist')));
 const PORT = 3000;
 
 // trips
-app.get('/tripp', (req, res) => {
-  let query;
-  if (!req.query) {
-    query = {};
-  } else {
-    query = req.query;
-  }
-  db.findTrip(query)
+app.get('/tripp/:id', (req, res) => {
+  db.findTrip(req.params.id)
```

Comment:
```suggestion
app.get('/tripp/:id', (req, res) => {
  db.findTrip(req.params.id)
```
These should be deleted.